src/__tests__/auth/socialAuth.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import {  shallow } from "enzyme";
+import {SocialAuth} from "../../components/auth/socialAuth";
+import userReducer from "../../reducers/authReducers/socialAuthReducer";
+import { SOCIAL_AUTH, LOGIN } from "../../actions/actionTypes";
+
+describe("socialAuth component", () => {
+  const wrapper = shallow(<SocialAuth/>);
+  it("it has a div with class social-btn text-center ", () => {
+    expect(wrapper.find("div").hasClass("social-btn text-center")).toBe(true);
+  
```

Comment:
Remove extra spaces

Reply:
@LehruAngela  when I write my tests without the "it" under describe, only the first test runs. The rest are skipped.